packages/react-composites/src/composites/CallComposite/pages/NoticePage.tsx

```diff
@@ -42,6 +45,8 @@ export function NoticePage(props: NoticePageProps): JSX.Element {
       data-ui-id={props.dataUiId}
       aria-atomic
     >
+      {/* @conditional-compile-remove(end-of-call-survey) */
+     <StarSurvey/>}
```

Comment:
This looks like this will always render no matter which kind of notice page this is. Should we be making sure that this is something that is only on the end call screen? this will show if you were kicked as well for example, is this something we want?

Reply:
I posted the question in the feature weekly, it feels like it makes more sense to show across the notice page? Calling wants to get as many data as possible :) but I am also not certain at a end user stand point this will be annoying

Reply:
the question to me for including it when the user is removed or disconnects is whether contoso would want feedback in those situations. Thinking about feedback from being removed from the call they might want to leave negative feedback that is "less than constructive." if we also look at it from the Teams perspective they only request feedback when the call ends naturally, if the end call reason was a disconnection the SDK team can report that.

Reply:
https://github.com/Azure/communication-ui-library/pull/3608/files Addressed this issue in this pr, survey is only shown on end call page now